### PR TITLE
Re-license as MIT

### LINKED ISSUE #454: Re-licensing covr as MIT
We are systematically re-licensing tidyverse and r-lib packages to use the MIT license, to make our package licenses as clear and permissive as possible. To do so, we need the approval of all copyright holders, which I have found by reviewing contributions from all all non-RStudio [contributors](https://github.com/r-lib/covr/graphs/contributors). @brodieG, @chen-liang-oracle, @chinqw, @daattali, @daroczig, @Delfic, @enbrown, @helske, @HenrikBengtsson, @hughjonesd, @jameslamb, @JohnMount, @jonclayden, @jpritikin, @karl-forner-quartz-bio, @katrinleinweber, @kenahoo, @kforner, @kirillseva, @kiwiroy, @maodunzhe, @MichaelChirico, @MishaCivey, @nibant, @patperry, @renkun-ken, @rmflight, @robertzk, @RolandASc, @siddharthab, @stevepeak, @surmann, @wamserma, @wligtenberg, would you permit us to re-license covr with the MIT license? If so, please comment "I agree" below.

* [x] [@chen-liang-oracle](https://github.com/r-lib/covr/commits?author=chen-liang-oracle)  
* [x] [@chinqw](https://github.com/r-lib/covr/commits?author=chinqw)  
* [x] [@daattali](https://github.com/r-lib/covr/commits?author=daattali)  
* [x] [@Delfic](https://github.com/r-lib/covr/commits?author=Delfic)  
* [x] [@helske](https://github.com/r-lib/covr/commits?author=helske)  
* [x] [@hughjonesd](https://github.com/r-lib/covr/commits?author=hughjonesd)  
* [x] [@jameslamb](https://github.com/r-lib/covr/commits?author=jameslamb)  
* [x] [@JohnMount](https://github.com/r-lib/covr/commits?author=JohnMount)  
* [x] [@jpritikin](https://github.com/r-lib/covr/commits?author=jpritikin)  
* [x] [@katrinleinweber](https://github.com/r-lib/covr/commits?author=katrinleinweber)  
* [x] [@kforner](https://github.com/r-lib/covr/commits?author=kforner)  
* [x] [@kiwiroy](https://github.com/r-lib/covr/commits?author=kiwiroy)  
* [x] [@maodunzhe](https://github.com/r-lib/covr/commits?author=maodunzhe)  
* [x] [@MichaelChirico](https://github.com/r-lib/covr/commits?author=MichaelChirico)  
* [x] [@MishaCivey](https://github.com/r-lib/covr/commits?author=MishaCivey)  
* [x] [@nibant](https://github.com/r-lib/covr/commits?author=nibant)  
* [x] [@patperry](https://github.com/r-lib/covr/commits?author=patperry)  
* [x] [@RolandASc](https://github.com/r-lib/covr/commits?author=RolandASc)  
* [x] [@siddharthab](https://github.com/r-lib/covr/commits?author=siddharthab)  
* [x] [@surmann](https://github.com/r-lib/covr/commits?author=surmann)  

(You may recall that we've been through this process once before in https://github.com/r-lib/covr/issues/256. We now understand the licensing landscape much better and are moving to make our policies consistent across the tidyverse)

### PR DESCRIPTION
**Pull request by batpigandme**
* Closes #454

### CODE REVIEW

File: .Rbuildignore [hunk 1]
```diff
@@ -32,3 +32,4 @@ _dev\.R$
 ^codecov\.yml$
 ^CODE_OF_CONDUCT\.md$
 ^\.github$
+^LICENSE\.md$
```
Comment (hadley) [inline .Rbuildignore:35]: I think you need to remove `^LICENSE$` higher up in the file

**Comment by hadley** [comment]
Hmmm, not sure why we're getting these failures.

**Comment by batpigandme** [comment]
Just tried moving it up even higher… It's not terribly high up in `.Rbuildignore` for other packages (e.g. in usethis https://github.com/r-lib/usethis/blob/57b109ab1e376d8fbf560e7e6adc19e0a04c5edd/.Rbuildignore#L12), so I'm not sure why that's causing those failures.

**Comment by hadley** [comment]
Sorry, I should've been more clear; it seems unlikely that it's your change causing these failures.

**Pull request merged**